File: Java Programing Basics/While-LoopExercise/src/OldBooks.java
import java.util.Scanner;

public class OldBooks {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String bookName = scanner.nextLine();
        int library = Integer.parseInt(scanner.nextLine());
        int counter = 0;
        boolean found = false;
        while (!found && counter < library){
            found = bookName.equals(scanner.nextLine());
            counter++;

        }
        if (found) System.out.printf("You checked %d books and found it.", --counter);
        else System.out.printf("The book you search is not here!%nYou checked %d books.", counter);

    }
}

File: Python/While_Loop(EXERCISE)/Old_Books.py
book_she_search = input()
library_capacity = int(input())
books = ""
counter = 0

while book_she_search != books and counter < library_capacity:
    books = input()
    counter += 1

if book_she_search == books:
    counter -= 1
    print(f"You checked {counter} books and found it.")

elif books != book_she_search or counter > library_capacity:
    print(f"The book you search is not here!")
    print(f"You checked {counter} books.")

